test/stdlib/IO_console_test.rb

```diff
@@ -0,0 +1,95 @@
+require_relative "test_helper"
+require "io/console"
+require "io/console/size"
+require 'pty'
+
+class IOConsoleSingletonTest < Test::Unit::TestCase
+  include TypeAssertions
+
+  library 'io-console'
+  testing "singleton(::IO)"
+
+  def test_io_console
+    assert_send_type "() -> File?",
+                     IO, :console
+    assert_send_type "(:close) -> nil",
+                     IO, :console, :close
+  end
+
+  def test_io_console_size
+    assert_send_type "() -> [Integer, Integer]",
+                     IO, :console_size
+  end
+
+  def test_io_default_console_size
+    assert_send_type "() -> [Integer, Integer]",
+                     IO, :default_console_size
+  end
+end
+
+class IOConsoleTest < Test::Unit::TestCase
+  include TypeAssertions
+
+  library 'io-console'
+  testing "::IO"
+
+  def helper
+    m, s = PTY.open
+  rescue RuntimeError
+    puts "no pty"
```

Comment:
```suggestion
    omit "no pty"
```

How about using `omit`? It clarifies the test is omitted.

Reply:
Sounds great!

I brought this code from https://github.com/ruby/io-console/blob/046e645407d150afc40b6fc67118200b8d932a72/test/io/console/test_io_console.rb#L410-L419 , but had to omit it because I had not checked how it would behave. 

Now that I've confirmed that `omit` works in this context, I'll use `omit`.